homeassistant/components/update/const.py

```diff
@@ -15,6 +15,7 @@ class UpdateEntityFeature(IntEnum):
     PROGRESS = 4
     BACKUP = 8
     RELEASE_NOTES = 16
+    AUTO_UPDATE = 32
```

Comment:
Thought of this a bit more. This should actually be an entity attribute. As the `supported_features` should not change at time.

This state reflects if auto-update is enabled on the device/service.

So probably should be just a

```py
    @property
    def auto_update(self) -> bool:
        """Indicates if the device or service has auto update enabled."""
        return self._attr_auto_update
```

Reply:
https://github.com/home-assistant/core/pull/69054